### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by palday**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,11 +49,13 @@ Typical distribution forms are _Bernoulli_ for binary data or _Poisson_ for coun
...
-|Linux|Ubuntu 18.04|x64|v1.4|1|
-|macOS|Catalina 10.15|x64|v1.4|1|
-|Windows|Server 2019|x64|v1.4|1|
-|Linux|Ubuntu 18.04|x86|v1.4|2|
-|Windows|Server 2019|x86|v1.4|2|
+|Linux|Ubuntu 18.04|x64|v1.4, 1.5|1|
+|macOS|Catalina 10.15|x64|v1.4, 1.5|1|
+|Windows|Server 2019|x64|v1.4, 1.5 |1|
+|Linux|Ubuntu 18.04|x86|  |2|
+|Windows|Server 2019|x86| |2|
```
Comment (palday) [inline README.md:56]:
```suggestion
```

**Comment by palday** [comment]
Actually, I'll let this be release PR, so I'll fix up the README on other things as well.

**Review by dmbates — APPROVED** [review]
That looks great!  Thanks.

**Comment by dmbates** [comment]
@palday  Feel free to squash and merge and tag a release.  It feels good to have this done!

**Pull request merged**